be/src/exprs/agg/group_concat.h

```diff
@@ -328,13 +328,15 @@ struct GroupConcatAggregateStateV2 {
             }
             data_columns->clear();
             data_columns.reset(nullptr);
+            delete output_index;
         }
     }
     // using pointer rather than vector to avoid variadic size
     // group_concat(a, b order by c, d), the a,b,',',c,d are put into data_columns in order, and reject null for
     // output columns a and b.
     std::unique_ptr<Columns> data_columns = nullptr;
     int output_col_num = 0;
+    uint64_t* output_index = new uint64_t(0);
```

Comment:
Why is `uint64_t*` used instead of `uint64_t`?

Reply:
it is also ok to use uint64_t.